verification/verification.py

```
@@ -524,3 +529,21 @@ async def on_verified_join(self, member: discord.Member):
+    @tasks.loop(seconds=60)
+    async def check_codes(self):
+        try:
+            await self.cur.execute("DELETE FROM `verification_codes` WHERE `generated_at` <= datetime('now', '-60 minutes')")
+            deleted = self.cur.rowcount
+            await self.con.commit()
+        except Exception:
+            await self.con.rollback()
+            logger.exception("Failed to purge expired verification codes")
+            return
+        if deleted > 0:
+            logger.info(f"[{arrow.now()}] Deleting {deleted} expired verification codes")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)